pallets/transaction-payment/README.md

```diff
@@ -0,0 +1,16 @@
+# Transaction Payment Pallet
+
+This pallet provides the basic logic needed to pay the absolute minimum amount needed for a
```

Comment:
@Leouarz @ToufeeqP 
Do we have somewhere a diff so that I can check what changes we have done?

Reply:
I will add a section in the README of this pallet for that